tests/unit/Endpoints/Webhooks/ProductHandlerTest.php

```diff
@@ -33,29 +34,7 @@ class ProductHandlerTest extends AbstractProductTest
 
     public function testCreateProduct()
     {
-        // Handle the product creation hook event
-        $original_options = $this->handle_hook_request(
-            self::CREATE_DATADUMP_DIRECTORY,
-            self::CREATE_DATADUMP_HOOK,
-            'ShopModule::ShopProduct',
-            'events'
-        );
-
-        // Retrieve the product from wordpress using the storekeeper id
-        $wc_products = wc_get_products(
-            [
-                'post_type' => 'product',
-                'meta_key' => 'storekeeper_id',
-                'meta_value' => $original_options['id'],
-            ]
-        );
-        $this->assertCount(
-            1,
-            $wc_products,
-            'Actual size of the retrieved product collection is not equal to one'
-        );
-
-        $wc_created_product = $wc_products[0];
+        $wc_created_product = $this->mockCreateProductRequestWithTest();
```

Comment:
can you add a test for images (i think that one goes wrong in the new implementation)?

* first sync product without images
* image is added to product in storekeeper
* new event is triggered and the image should be updated in woocomerce

Reply:
Added tests for images.